schematic/manifest/generator.py

```diff
@@ -33,6 +33,9 @@
     extract_component_validation_rules,
 )
 from schematic.utils.validate_utils import rule_in_rule_list
+from googleapiclient.errors import HttpError
+from opentelemetry.trace import Status, StatusCode
+from opentelemetry.trace import get_current_span
```

Comment:
Are the open telemetry imports used somewhere?

Reply:
I could remove them. I used them for experiment.